<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferParameters</class>
 <widget class="QDialog" name="TransferParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>520</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Характеристики передачі</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>423</width>
     <height>502</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="random_button">
      <property name="text">
       <string>Згенерувати випадкові дані</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" alignment="Qt::AlignHCenter">
     <widget class="QDialogButtonBox" name="dialog_button_box">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="packet_length_label">
          <property name="text">
           <string>Довжина пакета (байт)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="packet_length_spin_box">
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>127</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="error_probability_label">
          <property name="text">
           <string>Ймовірність помилки при передачі (%)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="error_probability_spin_box">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="retransmissions_number_label">
          <property name="text">
           <string>Найбільше число повторних передач</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="retransmissions_number_spin_box">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="delay_factor_label">
          <property name="text">
           <string>Середня величина коефіцієнта випадкової затримки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="delay_factor_spin_box">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="transfer_speed_label">
          <property name="text">
           <string>Швидкість передачі інформації (кбіт/с)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="transfer_speed_spin_box">
          <property name="minimum">
           <number>200</number>
          </property>
          <property name="maximum">
           <number>250</number>
          </property>
          <property name="value">
           <number>200</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="service_length_label">
          <property name="text">
           <string>Довжина службових полів пакета (байт)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="service_length_spin_box">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="transfer_radius_label">
          <property name="text">
           <string>Радіус передачі (м)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="transfer_radius_spin_box">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>200</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
